lib/tasks/webpacker/compile.rake

```diff
@@ -1,22 +1,26 @@
+require "open3"
 require "webpacker/env"
 require "webpacker/configuration"
-REGEX_MAP = /\A.*\.map\z/
 
 namespace :webpacker do
   desc "Compile javascript packs using webpack for production with digests"
   task compile: ["webpacker:verify_install", :environment] do
-    puts "Compiling webpacker assets 🎉"
+    new_line = "\n\n"
+    puts "[Webpacker] Compiling assets 🎉" + new_line
```

Comment:
`puts` implicitly adds a newline. Do we really want 3 total? I'd prefer to drop the `new_line` var and just use `puts` on its own line where we want spacing.